packages/swarm-components/src/Dropdown.jsx

```diff
@@ -0,0 +1,462 @@
+/* eslint-disable */
+import React, {
+	createContext,
+	Children,
+	useContext,
+	useState,
+	useEffect,
+	useRef,
+} from 'react';
+import Portal from '@reach/portal';
+import Rect, { useRect } from '@reach/rect';
+import WindowSize from '@reach/window-size';
+import { node, func, object, string, number, oneOfType, any } from 'prop-types';
+import { wrapEvent, checkStyles, assignRef } from '@reach/utils';
```

Comment:
Consider importing a single `import ReactUtils from '@reach/utils'` and then calling these methods from the parent object, e.g.

```js
ReachUtils.wrapEvent(...);
```

As with `React.x`, it can make the code easier to understand if the imported module context is explicit